concordium-consensus/src/Concordium/GlobalState/TransactionTable.hs

```diff
@@ -450,19 +445,20 @@ reversePTT trs ptt0 = foldr reverse1 ptt0 trs
 --  provided account address in the first component and the
 --  'Bool' in the second component is 'True' only if all transactions from the
 --  provided account are finalized.
+--  Returns @Nothing@ if no non-finalized transactions were recorded for the provided account.
 nextAccountNonce ::
     -- | The account to look up the next account nonce for.
     AccountAddressEq ->
     -- | The transaction table to look up in.
     TransactionTable ->
     -- | ("the next available account nonce", "whether all transactions from the account are finalized").
-    (Nonce, Bool)
+    Maybe (Nonce, Bool)
```

Comment:
**Minor thing:** Remove `Bool` from the return type.

It seems the boolean will always be `False` and is therefore not conveying any new information anymore.

Reply:
good idea.